### PR TITLE
Add core weekly 35

### PR DESCRIPTION
**Pull request by matks**

### CODE REVIEW

**Review by atomiix — APPROVED** [review]

File: news/_posts/core-weekly/2020-08-31-coreweekly-35-2020.md [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+* [#20724](https://github.com/PrestaShop/PrestaShop/pull/20724): Add test  'Default pack stock management'. Thank you [@nesrineabdmouleh](https://github.com/nesrineabdmouleh)
+* [#20692](https://github.com/PrestaShop/PrestaShop/pull/20692): Add tests 'Filter attributes' and 'Filter attribute values', by [@boubkerbribri](https://github.com/boubkerbribri)
+* [#20683](https://github.com/PrestaShop/PrestaShop/pull/20683): Fix faker data - Title's name should take at most 20 characters , by [@boubkerbribri](https://github.com/boubkerbribri)
+
+
+## Code changes in modules, themes & tools
+
+
+### Changes in developer documentation
+* [#691](https://github.com/PrestaShop/docs/pull/691): Add link to `_legacy_link` description.. Thank you [@likemusic](https://github.com/likemusic)
```
Comment (Progi1984) [inline news/_posts/core-weekly/2020-08-31-coreweekly-35-2020.md]:
```suggestion
* [#691](https://github.com/PrestaShop/docs/pull/691): Add link to `_legacy_link` description. Thank you [@likemusic](https://github.com/likemusic)
```

File: news/_posts/core-weekly/2020-08-31-coreweekly-35-2020.md [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+* [#690](https://github.com/PrestaShop/docs/pull/690): Allow installation of Hugo's mac os version via install script, by [@matthieu-rolland](https://github.com/matthieu-rolland)
+* [#689](https://github.com/PrestaShop/docs/pull/689): Fix some links, by [@matks](https://github.com/matks)
+* [#686](https://github.com/PrestaShop/docs/pull/686): Add a page to explain the UIKit use, by [@NeOMakinG](https://github.com/NeOMakinG)
+* [#685](https://github.com/PrestaShop/docs/pull/685): Scrollable tables.. Thank you [@likemusic](https://github.com/likemusic)
```
Comment (Progi1984) [inline news/_posts/core-weekly/2020-08-31-coreweekly-35-2020.md]:
```suggestion
* [#685](https://github.com/PrestaShop/docs/pull/685): Scrollable tables. Thank you [@likemusic](https://github.com/likemusic)
```

File: news/_posts/core-weekly/2020-08-31-coreweekly-35-2020.md [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+* [#684](https://github.com/PrestaShop/docs/pull/684): Add docker-compose.yml. Thank you [@likemusic](https://github.com/likemusic)
+* [#682](https://github.com/PrestaShop/docs/pull/682): Fix add() definition according to source phpDoc.. Thank you [@likemusic](https://github.com/likemusic)
+* [#681](https://github.com/PrestaShop/docs/pull/681): Remove unnecessary spaces which lead to code tag.. Thank you [@likemusic](https://github.com/likemusic)
```
Comment (Progi1984) [inline news/_posts/core-weekly/2020-08-31-coreweekly-35-2020.md]:
```suggestion
* [#682](https://github.com/PrestaShop/docs/pull/682): Fix add() definition according to source phpDoc. Thank you [@likemusic](https://github.com/likemusic)
* [#681](https://github.com/PrestaShop/docs/pull/681): Remove unnecessary spaces which lead to code tag. Thank you [@likemusic](https://github.com/likemusic)
```

File: news/_posts/core-weekly/2020-08-31-coreweekly-35-2020.md [hunk 4]
```diff
@@ -0,0 +1,157 @@
...
+* [#680](https://github.com/PrestaShop/docs/pull/680): Fix misprint. Thank you [@likemusic](https://github.com/likemusic)
+* [#679](https://github.com/PrestaShop/docs/pull/679): Correct typo on getting started part. Thank you [@Farelion](https://github.com/Farelion)
+* [#678](https://github.com/PrestaShop/docs/pull/678): Fix out of place question mark using.. Thank you [@likemusic](https://github.com/likemusic)
```
Comment (Progi1984) [inline news/_posts/core-weekly/2020-08-31-coreweekly-35-2020.md]:
```suggestion
* [#678](https://github.com/PrestaShop/docs/pull/678): Fix out of place question mark using. Thank you [@likemusic](https://github.com/likemusic)
```

File: news/_posts/core-weekly/2020-08-31-coreweekly-35-2020.md [hunk 5]
```diff
@@ -0,0 +1,157 @@
...
+* [#677](https://github.com/PrestaShop/docs/pull/677): Fix incorrect terms usage. Сlarify using `dump()`.. Thank you [@likemusic](https://github.com/likemusic)
+* [#676](https://github.com/PrestaShop/docs/pull/676): Fix server name from `Nginx` to `Apache`.. Thank you [@likemusic](https://github.com/likemusic)
```
Comment (Progi1984) [inline news/_posts/core-weekly/2020-08-31-coreweekly-35-2020.md]:
```suggestion
* [#676](https://github.com/PrestaShop/docs/pull/676): Fix server name from `Nginx` to `Apache`. Thank you [@likemusic](https://github.com/likemusic)
```

File: news/_posts/core-weekly/2020-08-31-coreweekly-35-2020.md [hunk 6]
```diff
@@ -0,0 +1,157 @@
...
+* [#675](https://github.com/PrestaShop/docs/pull/675): Fix path to download-links.png in localhost.md.. Thank you [@likemusic](https://github.com/likemusic)
```
Comment (Progi1984) [inline news/_posts/core-weekly/2020-08-31-coreweekly-35-2020.md]:
```suggestion
* [#675](https://github.com/PrestaShop/docs/pull/675): Fix path to download-links.png in localhost.md. Thank you [@likemusic](https://github.com/likemusic)
```

File: news/_posts/core-weekly/2020-08-31-coreweekly-35-2020.md [hunk 7]
```diff
@@ -0,0 +1,157 @@
...
+* [#685](https://github.com/PrestaShop/docs/pull/685): Scrollable tables.. Thank you [@likemusic](https://github.com/likemusic)
+* [#684](https://github.com/PrestaShop/docs/pull/684): Add docker-compose.yml. Thank you [@likemusic](https://github.com/likemusic)
+* [#682](https://github.com/PrestaShop/docs/pull/682): Fix add() definition according to source phpDoc.. Thank you [@likemusic](https://github.com/likemusic)
+* [#681](https://github.com/PrestaShop/docs/pull/681): Remove unnecessary spaces which lead to code tag.. Thank you [@likemusic](https://github.com/likemusic)
+* [#680](https://github.com/PrestaShop/docs/pull/680): Fix misprint. Thank you [@likemusic](https://github.com/likemusic)
+* [#679](https://github.com/PrestaShop/docs/pull/679): Correct typo on getting started part. Thank you [@Farelion](https://github.com/Farelion)
+* [#678](https://github.com/PrestaShop/docs/pull/678): Fix out of place question mark using.. Thank you [@likemusic](https://github.com/likemusic)
+* [#677](https://github.com/PrestaShop/docs/pull/677): Fix incorrect terms usage. Сlarify using `dump()`.. Thank you [@likemusic](https://github.com/likemusic)
```
Comment (Progi1984) [inline news/_posts/core-weekly/2020-08-31-coreweekly-35-2020.md]:
```suggestion
* [#677](https://github.com/PrestaShop/docs/pull/677): Fix incorrect terms usage. Сlarify using `dump()`. Thank you [@likemusic](https://github.com/likemusic)
```

**Review by Progi1984 — COMMENTED** [review]
Some typo. May be a bug from the coreweekly generator.

**Comment by matks** [comment]
> Some typo. May be a bug from the coreweekly generator.

No, it's because the PR title contains a final dot
https://github.com/PrestaShop/docs/pull/691 => "Add link to `_legacy_link` description."

So a nice feature on the CW generator would be "do not append a dot after the PR title if the PR titles already ends with a dot"

File: news/_posts/core-weekly/2020-08-31-coreweekly-35-2020.md (hunk 5 above)
Comment (Progi1984) [reply news/_posts/core-weekly/2020-08-31-coreweekly-35-2020.md]: @matks ;)

Comment (matks) [reply news/_posts/core-weekly/2020-08-31-coreweekly-35-2020.md]: We call him "eagle eye" !

**Review by Progi1984 — APPROVED** [review]
Not :eagle: :eye: but I know use the "Search" function of my browser :trollface:

**Comment by matks** [comment]
Thank you for review

**Pull request merged**